package.json

```diff
@@ -29,12 +29,12 @@
     "@percy/cli": "^1.10.2",
     "@percy/logger": "^1.10.2",
     "@percy/puppeteer": "^2.0.2",
-    "autoprefixer": "^9.8.6",
+    "autoprefixer": "^10.4.12",
     "body-parser": "^1.18.3",
-    "cheerio": "^1.0.0-rc.3",
+    "cheerio": "1.0.0-rc.3",
```

Comment:
I've had to pin this version

One of the package updates in this PR also allowed Cheerio to accept a compatible upgrade to **1.0.0-rc.12**

This broke lots of HTML escaping tests

Reply:
Not needed anymore, I've _only_ loaded the plugins mentioned via https://github.com/alphagov/govuk-frontend/issues/2469#issuecomment-1006658228